projects/plugins/jetpack/json-endpoints/jetpack/class.jetpack-json-api-plugins-install-endpoint.php

```diff
@@ -105,6 +105,7 @@ protected function install() {
 		}
 
 		// No errors, install worked. Now replace the slug with the actual plugin id
+		// @phan-suppress-next-line PhanPossiblyUndeclaredVariable -- we return early if there is an issue; otherwise $index and $slug are set in the foreach loop
```

Comment:
This code looks suspicious anyway. Why does only the last entry processed by the loop get this assignment?

Probably that's not really a matter for *this* PR, but I'd at least note that as a todo comment.

Reply:
378840a2a4